src/cli.rs

```diff
@@ -531,9 +531,11 @@ pub struct Opt {
     #[structopt(long = "line-fill-method")]
     pub line_fill_method: Option<String>,
 
-    /// The width of underline/overline decorations. Use --width=variable to extend decorations and
-    /// background colors to the end of the text only. Otherwise background colors extend to the
-    /// full terminal width.
+    /// The width of underline/overline decorations. Can be absolute, absolute [+|-] offset, or
+    /// only an offset - in the last case the detected terminal width is changed
+    /// by the offset. Examples: "72", "80-2", "-2" or "98+2".
```

Comment:
Can you remind me what a common use case for `+` is?

Reply:
Honestly, none that I can think of - feature completeness? Making sure the ALU still knows how to add?

Reply:
Haha :) okay, I'm just thinking about how to make it as easy as possible to understand. What do you think about something like

```
 The width of underline/overline decorations. Examples: "72", "-2" (auto-detected terminal width minus 2). An expressions such as "74-2" is also valid (equivalent to 72 but may be useful in shell programming if a variable exists holding the value "74").
```